README.md

```diff
@@ -223,37 +230,35 @@ You can toggle the presenter or overview mode by pressing respectively `alt+p` a
 
 ## Fullscreen
 
-Fullscreen can be toggled via browser options, or by **hovering over the bottom right corner of your window until the fullscreen icon appears and clicking it**.
+Fullscreen can be toggled via browser options, or by hovering over the bottom right corner of your window until the fullscreen icon appears and clicking it.
 
 <a name="pdf-export"></a>
 
 ## PDF Export
 
-Exporting a totally sweet looking PDF from your totally sweet looking Spectacle presentation is absurdly easy. You can either do this via the browser, or from the command line:
+You can export a PDF from your Spectacle presentation either from the command line or browser:
 
 #### CLI
 
 - Run `npm install spectacle-renderer -g`
 - Run `npm start` on your project and wait for it to build and be available
 - Run `spectacle-renderer`
-- A totally cool PDF is created in your project directory
 
-For more options and configuration of this tool, check out:
+A PDF is created in your project directory. For more options and configuration of this tool, check out:
 
 [https://github.com/FormidableLabs/spectacle-renderer](https://github.com/FormidableLabs/spectacle-renderer)
 
 #### Browser
 
-- Run `npm start`
-- Open [http://localhost:3000/](http://localhost:3000/)
-- Add `export&` after the `?` on the URL of page you are redirected to, e.g.: [http://localhost:3000/#/?export&\_k=wbyhif](http://localhost:3000/#/?export&_k=wbyhif)
+After running `npm start` and opening [http://localhost:3000/#/](http://localhost:3000/#/) in your browser...
+
+- Add `?export&_k=wbyhif` after the `/` on the URL of page you are redirected to, e.g.: [http://localhost:3000/#/?export&\_k=wbyhif](http://localhost:3000/#/?export&_k=wbyhif)
 - Bring up the print dialog `(ctrl or cmd + p)`
-- Check "Background Graphics" to on if you are about that life
 - Change destination to "Save as PDF", as shown below:
 
 ![http://i.imgur.com/t6GL5Oc.png](http://i.imgur.com/t6GL5Oc.png)
 
-If you want to print your slides, and want a printer friendly version, simply repeat the above process but instead print from [http://localhost:3000/?export&print](http://localhost:3000/?export&print)
+If you want a printer friendly version, repeat the above process but instead print from [http://localhost:3000/#/?export&print](http://localhost:3000/#/?export&print)
```

Comment:
It might be handy to have some kind of markdown chart with the available query parameters for these various features, as a summary somewhere in this doc

i.e.

Query | Description
------ | ------------
`?export` | Creates a single-page overview of your slides, that you can then print
`?export&print` | Creates a black & white single-page overview of your slides
`...` | Another cool feature that we offer